simpleflow/activity.py

```diff
@@ -87,7 +87,25 @@ def name(self):
         return '.'.join([prefix, name])
 
     def __repr__(self):
-        return 'Activity(name={}, version={}, task_list={})'.format(
+        return 'ActivityType(name={}, version={}, task_list={})'.format(
             self.name,
             self.version,
             self.task_list)
+
+
+class Activity(object):
```

Comment:
Yes I thought the same thing, but `ActivityTask` is generated by the executor, and we need a object containing the type + the args before the submission.